### DISCUSSION TITLE
OCR cannot detect arabic numbers

### DISCUSSION THREAD

**Discussion by mariojava** [adaptech-cz/Tesseract4Android#30] [Q&A]
Hello!

I'm trying to extract arabic text and numbers as well from ID card.
Can you suggest on what traineddata to use and how to initialize the Tesseract base API - OEM_DEFAULT or? 
Currently I'm using these ones and I'm extracting only the text with good quality (without the numbers):

"ara.traineddata";
"eng.traineddata";
"ara.cube.bigrams";
"ara.cube.fold";
"ara.cube.lm";
"ara.cube.nn";
"ara.cube.params";
"ara.cube.size";
"ara.cube.word-freq";

**Comment by Robyer** [comment]
Hi, since Tesseract4Android is just wrapper around Tesseract library, for these questions you should go to their main repository and read documentation, mailing list, or ask question on their user mailing list, as stated here: https://github.com/tesseract-ocr/tesseract#support

If you want to know how to use Tesseract4Android API, you can look at https://github.com/adaptech-cz/Tesseract4Android/blob/master/tesseract4android/src/androidTest/java/com/googlecode/tesseract/android/TessBaseAPITest.java for inspiration.
